### PR TITLE
docs: TECH-355 adds instruction on the README file on how to run the capture app

### PR DESCRIPTION
**Pull request by paschalidi**
Hello @JoakimSM 👋  Ticket is https://jira.dhis2.org/browse/TECH-355

I added some text on the README.md to explain the steps I followed for running the capture-app locally. 

I hope this way when someone else looks into it will be easy for them to follow the steps and have the app running on their environment. 

As always your comments and thoughts are more than welcomed!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2170 +1,78 @@
...
-
-With this setup Netlify will build and deploy when you push to git or open a pull request:
-
-1. [Start a new netlify project](https://app.netlify.com/signup)
-2. Pick your Git hosting service and select your repository
-3. Click `Build your site`
+#### 3. Set up the API base url
 
-**Support for client-side routing:**
+For the application to fetch data from the backend we need to set up a base url. Make sure in your `.env.development` file the following is included:
```
Comment (JoakimSM) [inline README.md:52]:
```suggestion
The application needs to communicate with a DHIS2 backend instance. The `.env.development` file contains default configuration, but you can override it by supplying a `.env.development.local` file in the root folder of the project.
```

File: README.md [hunk 2]
```diff
@@ -1,2170 +1,78 @@
...
-Choose `build` as the path to deploy.
-
-**To setup continuous delivery:**
-
-With this setup Netlify will build and deploy when you push to git or open a pull request:
-
-1. [Start a new netlify project](https://app.netlify.com/signup)
-2. Pick your Git hosting service and select your repository
-3. Click `Build your site`
+#### 3. Set up the API base url
```
Comment (JoakimSM) [inline README.md:50]:
```suggestion
#### 3. Set environment variables for backend communication
```

File: README.md [hunk 3]
```diff
@@ -1,2170 +1,78 @@
...
-By default `npm test` runs the watcher with interactive CLI. However, you can force it to run tests once and finish the process by setting an environment variable called `CI`.
-
-When creating a build of your application with `npm run build` linter warnings are not checked by default. Like `npm test`, you can force the build to perform a linter warning check by setting the environment variable `CI`. If any warnings are encountered then the build fails.
-
-Popular CI servers already set the environment variable `CI` by default but you can do this yourself too:
+# Capture app
 
-### On CI servers
-#### Travis CI
+The Capture app allows you capture, modify and list tracked entity instances with its enrollments and events. The Tracker Capture app works with multiple event-based programs in DHIS2, which handles events linked to registered entities. These programs are suitable for handling disease programmes where multiple visits is required. The Capture app can be used for disease programmes such as tubercolosis and malaria. However the app is not tied to any specific domain and can potentially be utilized for any scenario of multiple event-based information.
```
Comment (JoakimSM) [inline README.md:3]: The first part is not true as we don't have tracker functionality yet. Maybe you can get some inspiration from the dhis2 capture app docs?

Comment (paschalidi) [reply README.md:3]: ah i copied it from the docs. probably wrong docs. :P

**Comment by paschalidi** [comment]
@JoakimSM really good points! Thanks 👍 Have another look please

File: README.md [hunk 4]
```diff
@@ -1,2170 +1,82 @@
...
-
-### Surge
-
-Install the Surge CLI if you haven’t already by running `npm install -g surge`. Run the `surge` command and log in you or create a new account.
-
-When asked about the project path, make sure to specify the `build` folder, for example:
-
-```sh
-       project path: /path/to/project/build
+REACT_APP_DHIS2_BASE_URL="https://debug.dhis2.org/2.34dev/"
```
Comment (JoakimSM) [inline README.md:59]: And also maybe do something with these, remove them or make a complete example with all of the enviroment variables. I think it's fine removing them

Comment (paschalidi) [reply README.md:59]: I'd prefer to make a complete example of a file so that someone completely new can run the app. 

What you think?

Comment (JoakimSM) [reply README.md:61]: Ok, that's fine.

**Comment by paschalidi** [comment]
Thanks for the review both! :D

**Pull request merged**